### PR TITLE
Process manager

### PR DESCRIPTION
**Pull request by estebanprimost**
- Use PM2 for start/stop/restart daemon
- Add `repl` as command with PM2

### CODE REVIEW

File: packages/cli/src/commands/start.js [hunk 1]
```diff
@@ -1,36 +1,68 @@
...
 
 const { flags } = require('@oclif/command')
+const pify = require('pify')
+const pm2 = require('pm2')
+
+const ReplCommand = require('./repl')
 
 const BaseCommand = require('../base-command')
 
+const SEEDER_DAEMON = 'seeder-daemon'
```
Comment (dpaez) [inline packages/cli/src/commands/start.js]: export process name, so we can use it from other places, like `stop` command.

File: packages/cli/src/commands/stop.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+const pm2Connect = pify(pm2.connect.bind(pm2))
+const pm2Disconnect = pify(pm2.disconnect.bind(pm2))
+const pm2Stop = pify(pm2.stop.bind(pm2))
+
+class StopCommand extends BaseCommand {
+  async run () {
+    await pm2Connect()
+
+    await pm2Stop('seeder-daemon')
```
Comment (dpaez) [inline packages/cli/src/commands/stop.js]: use imported process name.

**Review by dpaez — APPROVED** [review]
Looks good! 💪 Just left some comments.

File: packages/cli/src/commands/stop.js (hunk 2 above)
Comment (estebanprimost) [reply packages/cli/src/commands/stop.js]: Done, thanks!

File: packages/cli/src/commands/stop.js [hunk 3]
```diff
@@ -1,6 +1,8 @@
 const pify = require('pify')
 const pm2 = require('pm2')
 
+const { SEEDER_DAEMON } = require('./start')
```
Comment (dpaez) [inline packages/cli/src/commands/stop.js:4]: Looks good to me, if more variables are needed in the future we can come up with a new `constants.js` file or something like that.

**Review by dpaez — APPROVED** [review]
done weird film

**Pull request merged**